packages/ui/Scrollable/Scrollable.js

```diff
@@ -385,13 +399,17 @@ class ScrollableBase extends Component {
 			{hasDataSizeChanged} = this.childRef;
 
 		// Need to sync calculated client size if it is different from the real size
-		if (this.childRef.syncClientSize) {
+		const isSync = this.childRef.syncClientSize ? this.childRef.syncClientSize() : null;
+
+		if (!this.initialScrollTo && this.props.initialScrollPosition) {
+			const {left, top} = this.props.initialScrollPosition;
+			this.scroll(left, top);
```

Comment:
If this routine does not forward 'onScrollStart', it looks better to use `start` instead of `scroll`.